app/controllers/application_controller.rb

```diff
@@ -1,2 +1,18 @@
 class ApplicationController < ActionController::API
+  AUTHORIZATION_HEADER = 'Authorization'.freeze
+
+  include Pundit
+  include ExceptionsHandler
+
+  def current_user
+    header = request.headers[AUTHORIZATION_HEADER]
+    token = header.split(' ').last if header
+    @current_user ||= Users::GetCurrentUserService.new(token).call
+  end
+
+  def load_and_authorize_resource
```

Comment:
try not to use that header. it's cancancan ability. try to use just pundit for that